### PR TITLE
Pioneer 14a

### PR DESCRIPTION
**Pull request by cdobs**
Updated CruiseInformation.csv with Pioneer 14a cruise

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -273,6 +273,7 @@ AJ20200506,R/V Adolf Jensen,2020-05-06T12:00:00,2020-05-08T12:00:00,Irminger gli
 TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries
 ZP20200604,R/V Zephyr,2020-06-04T08:00:00,2020-06-08T23:59:59,Papa Glider gp_537 and gp_575 recoveries
 AR44,R/V Neil Armstrong,2020-06-07T00:00:00,2020-06-17T23:59:00,Pioneer 14
+TS20200613,F/V Tenacious II, 2020-06-13T11:00:00,2020-06-14T04:00:00,Pioneer 14a Glider deployments
 AR46,R/V Neil Armstrong,2020-08-08T00:00:00,2020-09-06T23:59:00,Irminger 7
 AR48,R/V Neil Armstrong,2020-09-30T00:00:00,2020-10-20T23:59:00,Pioneer 15
 TN382,R/V Thomas G. Thompson,2020-08-02T00:00:00,2020-08-28T23:59:59,Cabled (rvdata)
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:279]: looks good

Comment (snwhite) [reply cruise/CruiseInformation.csv:279]: Loods good.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -179,8 +179,8 @@ CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00583,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,583,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
-CGCON-GLDRGP-00470,,Sensor,1,Mobile Asset Controller for Profiling Glider,,Teledyne Webb,G2 Slocum Gliders,470,,,,TO BE DELETED,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00583,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL583 Controller,Teledyne Webb,G2 Slocum Gliders,583,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:182]: Looks good

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -179,8 +179,8 @@ CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,
...
+CGCON-GLDRGP-00470,,Sensor,1,Mobile Asset Controller for Profiling Glider,PG470 Controller,Teledyne Webb,G2 Slocum Gliders,470,,,,TO BE DELETED,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:183]: Looks good

File: bulk/node_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -1,51 +1,90 @@
...
 CGVEH-GLDRCP-00380,A00368,Node,1,Coastal Glider 380,COASTAL GLIDER 1000M,Teledyne Webb,DG-1000,380,,20130806,186935,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00387,A00585,Node,1,Coastal Glider 387,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,387,,20131223,186935,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00388,A00583,Node,1,Coastal Glider 388,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,388,,20131223,186935,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00389,A00584,Node,1,Coastal Glider 389,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,389,,20131223,186935,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00514,,Node,1,Coastal Glider 514,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,514,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00564,,Node,1,Coastal Profiling Glider 564,COASTAL GLIDER 200M,Teledyne Webb,ASSY 302392-3,564,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00583,,Node,1,Coastal Profiling Glider 583,COASTAL GLIDER 200M,Teledyne Webb,ASSY 302392-3,583,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/node_bulk_load-AssetRecord.csv:65]: Looks good

File: bulk/node_bulk_load-AssetRecord.csv [hunk 5]
```diff
@@ -54,67 +93,30 @@ CGVEH-GLDROO-00477,A00964,Node,1,Open Ocean Glider 477,OPEN OCEAN GLIDER 1000M,T
...
-ATAPL-71533-00002,ATAPL-71533-00002,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00002,,,,,University of Washington
-ATAPL-71533-00003,ATAPL-71533-00003,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00003,,,,,University of Washington
-ATAPL-71533-00004,ATAPL-71533-00004,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00004,,,,,University of Washington
-ATAPL-71533-00005,ATAPL-71533-00005,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00005,,,,,University of Washington
+CGVEH-GWFPGX-77401,A01080,Node,0,Global Wire-Following Profiler,GLOBAL WIRE FOLLOWING PROFILER,McLane,MMP-210-3512,ML12774-01,,20140925,98735.1,,Woods Hole Oceanographic Institution
+CGVEH-GWFPGX-93602,A00267,Node,0,Global Wire-Following Profiler,WFP GLOBAL,McLane,MMP-210-3512,ML12936-02,,20130320,111181.36,,Woods Hole Oceanographic Institution
+CGVEH-GWFPGX-99701,A00268,Node,0,Global Wire-Following Profiler,WFP GLOBAL,McLane,MMP-210-3512,ML12997-01,,20130320,97150.52,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/node_bulk_load-AssetRecord.csv:122]: I don't see "CGVEH-GLDROO-00470" in the preview that Github shows me for file changes. But I confirmed it was there by clicking "view file" and it's on line 91. Looks good.

File: bulk/node_bulk_load-AssetRecord.csv (hunk 4 above)
Comment (rtravis-whoi) [reply bulk/node_bulk_load-AssetRecord.csv:65]: looks good

File: bulk/node_bulk_load-AssetRecord.csv (hunk 5 above)
Comment (rtravis-whoi) [reply bulk/node_bulk_load-AssetRecord.csv:121]: Same here - weird!

File: cruise/CruiseInformation.csv [hunk 6]
```diff
@@ -272,7 +272,8 @@ WJ20200323,M/V Warren Jr.,2020-03-23T00:00:00,2020-03-27T23:59:00,Pioneer 13c AU
 AJ20200506,R/V Adolf Jensen,2020-05-06T12:00:00,2020-05-08T12:00:00,Irminger glider gi_525 recovery
 TS20200522,F/V Tenacious II,2020-05-22T06:00:00,2020-05-22T23:59:59,Pioneer 13b glider recoveries
 ZP20200604,R/V Zephyr,2020-06-04T08:00:00,2020-06-08T23:59:59,Papa Glider gp_537 and gp_575 recoveries
-AR44,R/V Neil Armstrong,2020-06-07T00:00:00,2020-06-17T23:59:00,Pioneer 14
+AR44,R/V Neil Armstrong,2020-06-07T16:26:00,2020-06-16T16:32:00,Pioneer 14
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:275]: Looks good!

**Pull request merged**